source/sass/components/_button.scss

```diff
@@ -7,6 +7,17 @@
     &:hover {
         cursor: pointer;
     }
+
+    &--primary {
+        background-color: $color--primary;
+        color: $color--white;
+        transition: background-color $transition;
+
+        &:hover {
```

Comment:
does this need `:focus` & `:active` as well or are they defined elsewhere?

Reply:
Needed here, added